convert/lib/convert.py

```diff
@@ -0,0 +1,545 @@
+                ["install", "overrides", "external_dns", "domain_filters"],
+                cdkconfig,
+            )[0],
+            "efs_backups": cdkconfig["efs"]["backup"]["enable"],
+            "efs_backup_schedule": cdkconfig["efs"]["backup"]["schedule"],
+            "efs_backup_cold_storage_after": cdkconfig["efs"]["backup"]["move_to_cold_storage_after"],
+            "efs_backup_delete_after": cdkconfig["efs"]["backup"]["delete_after"],
+            "efs_backup_force_destroy": cdkconfig["efs"]["backup"]["removal_policy"] == "DESTROY",
+            "eks_custom_role_maps": eks_custom_role_maps,
+        }
+
+        print(json.dumps(tfvars))
+
+    def clean_stack(self):
+        self.setup(full=True, no_stacks=self.args.resource_file)
+
+        if self.args.resource_file:
+            with open(self.args.resource_file) as f:
+                self.stacks = yaml.safe_load(f.read())
+
+        include_types = self.args.include_types or clean_categories
+
+        for t in include_types:
+            if t not in clean_categories:
+                raise ValueError(f"{t} not a valid category for `--include-types`. Should be: {clean_categories}")
+
+        lambda_safe = [
+            cdk_ids.lambda_function.value,
+            cdk_ids.iam_role.value,
+            cdk_ids.lambda_layerversion.value,
+            cdk_ids.stepfunctions_statemachine.value,
+        ]
+        nukey = {
+            "efs_stack": {
+                "(LogRetention|backuppostcreationtasks).*": lambda_safe,
+            },
+            "eks_cluster_stack": {
+                "(LogRetention|IsCompleteHandler|NodeProxyAgentLayer|OnEventHandler|Provider).*": lambda_safe,
+            },
+            "eks_kubectl_stack": {
+                "(Handler|KubectlLayer|ProviderframeworkonEvent).*": lambda_safe,
+            },
+            "eks_stack": {
+                f"(snapshot|{self.stack_name}ebscsi|{self.stack_name}DominoEcrRestricted|autoscaler)": [
+                    cdk_ids.iam_policy.value
+                ],
+                f"(eksMastersRole|{self.stack_name}NG)$": [cdk_ids.iam_role.value],
+                "eksKubectlReadyBarrier": [cdk_ids.ssm_parameter.value],
+                "(clusterpost(creation|deletion)tasks|LogRetention)": lambda_safe,
+                "Unmanaged": [cdk_ids.instance_profile.value, cdk_ids.asg.value, cdk_ids.launch_template.value],
+            },
+            "s3_stack": {
+                "CustomS3AutoDeleteObjectsCustomResourceProvider": lambda_safe,
+            },
+            "vpc_stack": {
+                "endpointssg": [cdk_ids.security_group.value],
+                "bastion": [
+                    cdk_ids.instance.value,
+                    cdk_ids.instance_profile.value,
+                    cdk_ids.iam_role.value,
+                    cdk_ids.eip.value,
+                ],
+                "VPCrejectFlowLogsFlowLog": [cdk_ids.flowlog.value],  # TODO: should this be in the module?
```

Comment:
FWIW to answer the question, I think yes? Why not import it?

Reply:
I'll make a ticket to add the functionality to terraform-aws-eks. I'll keep this here for the sake of tracking and update it in tandem.